cmd/grr/workflow.go

```diff
@@ -210,3 +223,7 @@ func providersCmd(config grizzly.Config) *cli.Command {
 	}
 	return cmd
 }
+
+func getJsonnetFolders() []string {
```

Comment:
Can we make it this:
```suggestion
func getDefaultJsonnetFolders() []string {
```

Then I think we can merge this.

Reply:
Done :)

Reply:
@malcolmholmes Done :)